setup.py

```diff
@@ -1,23 +1,15 @@
 # encoding=utf8
-import os
-from distutils.core import setup
 
-def read(fname):
-    return open(os.path.join(os.path.dirname(__file__), fname)).read()
-
-README = read('README.rst')
+from setuptools import setup
 
 setup(
-    name = "django-form-designer",
-    version = "0.8.0.5",
-    url = 'http://github.com/philomat/django-form-designer',
-    license = 'BSD',
-    description = "Design contact forms, search forms etc from the Django admin, without writing any code. Integrates with Django CMS.",
-    long_description = README,
-
-    author = 'Samuel Luescher',
-    author_email = 'sam at luescher dot org',
-    packages = [
+    name='django-form-designer-ai',
+    version='0.9.0',
+    url='http://github.com/andersinno/django-form-designer-ai',
+    license='BSD',
+    maintainer='Aarni Koskela',
+    maintainer_email='aarni@anders.fi',
```

Comment:
Maintainer should be Anders Innovations (with some general email that we have), since there will be also other Anders developers involved.